include/zephyr/net/ieee802154_radio.h

```diff
@@ -169,6 +169,12 @@ enum ieee802154_config_type {
 	/** Sets the current MAC frame counter value for radios supporting transmit security. */
 	IEEE802154_CONFIG_FRAME_COUNTER,
 
+	/** Sets the current MAC frame counter value if current counter is greater than passed value
```

Comment:
```suggestion
	/** Sets the current MAC frame counter value if the provided value is greater than the current value
```

Reply:
I reworded it slightly differently than suggested.